app/assets/v2/js/pages/bounty_details.js

```diff
@@ -429,25 +429,27 @@ var show_interest_modal = function() {
       modal.on('submit', function(event) {
         event.preventDefault();
 
-        var has_question = event.target[0].value;
-        var issue_message = event.target[2].value;
+        var issue_message = event.target[0].value;
+        var agree_precedence = event.target[1].checked;
+        var agree_not_to_abandon = event.target[2].checked;
 
-        var agree_precedence = event.target[3].checked;
-        var agree_not_to_abandon = event.target[4].checked;
+        if (!issue_message) {
+          _alert({message: 'Please provide an action plan for this ticket.'}, 'error');
```

Comment:
Should we `gettext()` these message strings?

Reply:
we should :P

Reply:
😭

Reply:
:trollface: 
![eyeofsauron](https://user-images.githubusercontent.com/5358146/39827032-4fc4c794-53d4-11e8-8ae7-afbc8bd8d65e.jpg)